react/package.json

```diff
@@ -35,10 +35,10 @@
     "ramda": "^0.26.1",
     "react-apollo": "^3.1.3",
     "typescript": "3.9.7",
-    "vtex.checkout-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-graphql@0.50.0/public/@types/vtex.checkout-graphql",
-    "vtex.checkout-resources": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-resources@0.40.0/public/@types/vtex.checkout-resources",
+    "vtex.checkout-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-graphql@0.59.0/public/@types/vtex.checkout-graphql",
+    "vtex.checkout-resources": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-resources@0.42.0/public/@types/vtex.checkout-resources",
```

Comment:
Could you re-run the setup typings command? I've released `vtex.checkout-resources@0.45.0` (your other PR), and I'm not sure why it was outdated for you 🤔 , but it should work now.

Reply:
Done!